images/scripts/lib/build-deps.sh

```diff
@@ -20,4 +19,7 @@ curl -s https://raw.githubusercontent.com/cockpit-project/cockpit/$branch/tools/
     sed 's/.*/"&"/' |
     tr '\n' ' '
 
-echo "$stable_branch_deps"
+# HACK: Install selinux-policy manually as it will be pulled in through dependencies only once https://github.com/cockpit-project/cockpit/pull/15707 lands
+if [ "${1#fedora*}" != "$1" ] || [ "${1#rhel*9}" != "$1" ]; then
+    echo '"selinux-policy" "selinux-policy-devel"'
```

Comment:
Curious syntax with the double quotes -- we did not need that before, like in commit eb22f708b6bfd? but either way, if it works, that's fine.

Reply:
Output of it looks like this:
```
"/usr/bin/python3" "autoconf" "automake" "docbook-style-xsl" "gcc" "gdb" "gettext >= 0.19.7" "glib-networking" "glib2-devel >= 2.50.0" "gnutls-devel >= 3.4.3" "krb5-devel >= 1.11" "krb5-server" "libssh-devel >= 0.8.5" "libxslt-devel" "make" "openssh-clients" "openssl-devel" "pam-devel" "pcp-libs-devel" "pkgconfig(gio-unix-2.0)" "pkgconfig(json-glib-1.0)" "pkgconfig(polkit-agent-1) >= 0.105" "sed" "systemd-devel >= 235" "xmlto" "zlib-devel" "selinux-policy" "selinux-policy-devel"
```

Those quotes are there because it may contain version number and while I am aware that we don't need it in this case, I wanted to keep it consistent.